### PR TITLE
Week3 restaurants

### PR DESCRIPTION
**Pull request by Jonathanlouiskirk**

### CODE REVIEW

File: Jonathan/restaurants_challenge.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+print(restaurant)
+
+# TODO: Write code to print the latitude and longitude of Four Barrel Coffee.
+# TODO: Write code to print the complete address of the Four Barrel Coffee, formatted as a string - it should include the address, city, state and the zip code.
+# TODO: Write code to print the URL of the website of Four Barrel Coffee.
+
+
+print(f'The latitude is {resturants["latitude"]}')
+print(f'The longitude is {resturants["latitude]}')
```
Comment (waldenwang9966) [inline Jonathan/restaurants_challenge.py:41]: Take a look at line 41. Notice any mistake?

File: Jonathan/restaurants_challenge.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+print()
+
+
+print("Question 5")
+'''
+Printing out all 3 of our restaurants every time is getting annoying. Let's clean up our code!
+'''
+
+# TODO: Put your 3 restaurant dictionaries into a list called `restaurants`
+# TODO: Loop through your list and print out the name and address of each restaurant
```
Comment (waldenwang9966) [inline Jonathan/restaurants_challenge.py:93]: Looks like you forgot question 3 to 5.

File: Jonathan/stocks_challenge 2.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+'''
+
+print()
+
+print("Challenge 3.2.3: Output for the user the result")
+# TODO: Once you have calculated the number of stocks that can be purchased, print the result for the client. Result should be in a format like this:
+
+# Alex has $5000 in savings and he can buy 50 shares of Apple at the current price of $100.
+
+print()
```
Comment (waldenwang9966) [inline Jonathan/stocks_challenge 2.py:39]: Is this an old file?

**Review by waldenwang9966 — CHANGES_REQUESTED** [review]

File: Jonathan/stocks_challenge 2.py (hunk 3 above)
Comment (Jonathanlouiskirk) [reply Jonathan/stocks_challenge 2.py:39]: it must be sorry

File: Jonathan/restaurants_challenge.py (hunk 1 above)
Comment (Jonathanlouiskirk) [reply Jonathan/restaurants_challenge.py:41]: i fixed it

Comment (Jonathanlouiskirk) [inline Jonathan/restaurants_challenge.py:41]: needs a ' after latitude

**Pull request closed without merging**